### ISSUE TITLE
Move search results to an ordered list 

### ISSUE THREAD

**Issue by zackkrida** [WordPress/openverse#794]
Openverse search results should be wrapped in an ordered list and  `li` element for each returned result. This will help screen reader users identify the number of results returned and their current position in the results.

**Comment by dhruvkb** [comment]
@zackkrida why unordered? Assuming we are sorting results by relevance (or in the future, `indexed_on`) the order of the results is meaningful information. `<ol>` would be more appropriate imo.

**Comment by zackkrida** [comment]
@dhruvkb 👍 you're right. I use ordered lists so infrequently in HTML that I didn't even consider it.
